### PR TITLE
Russian Translation of React Projects.

### LINKED ISSUE #6808: Russian Translation of React Projects
- [x] Build a Markdown Previewer
- [x] Build a Camper Leaderboard
- [x] Build a Recipe Box
- [x] Build the Game of Life
- [x] Build a Roguelike Dungeon Crawler Game

### PR DESCRIPTION
**Pull request by rfprod**
Closes #6808

### CODE REVIEW

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 1]
```diff
@@ -27,6 +27,20 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте предпросмотрщик облегченного языка разметки Markdown",
+      "descriptionRu": [
+        "<strong>Задание:</strong> Создайте приложение <a href='http://codepen.io' target='_blank'>CodePen.io</a>, функционал которого схож с этим: <a href='http://codepen.io/FreeCodeCamp/full/obYYqW' target='_blank'>http://codepen.io/FreeCodeCamp/full/obYYqW</a>.",
+        "<strong>Правило #1:</strong> Не подсматривайте в код приложения-примера. Разберитесь самостоятельно.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `Не подсматривайте` **в** `код` -> `Не подсматривайте код`, here and everywhere.
`Разберитесь самостоятельно` sounds rude. Needs to be replaced with something like this: `Напишите его самостоятельно` here and everywhere.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 2]
```diff
@@ -27,6 +27,20 @@
...
+        "<strong>Правило #2:</strong> Приложение должно удовлетворять нижеприведённым <a href='http://en.wikipedia.org/wiki/User_story' target='_blank'>пользовательским историям</a>. Используйте любые библиотеки или API, которые потребуются. Придайте ему свой личный стиль.",
+        "<strong>Правило #3:</strong> Для создания этого проекта вы должны использовать Sass и React.",
+        "<strong>Пользовательская История:</strong> Я могу использовать облегчённый язык разметки в стиле GitHub в текстовом поле.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `Пользовательская `**И**`стория` -> `Пользовательская `**и**`стория`, here and everywhere.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 3]
```diff
@@ -27,6 +27,20 @@
...
+        "<strong>Пользовательская История:</strong> Я могу видеть предварительный просмотр вывода моей облегченной разметки по мере ввода текста.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: IMHO there is no need to use the word `облегченной` or similar one every time.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 4]
```diff
@@ -27,6 +27,20 @@
...
+        "<strong>Подсказка:</strong> Вам не нужно интерпретировать облегчённую разметку самостоятельно - вы можете импортировать библиотеку Marked для этого: <a href='https://cdnjs.com/libraries/marked'>https://cdnjs.com/libraries/marked</a>",
+        "<strong>Заметка:</strong> Если вы хотите использовать синтаксис React JSX, вам понадобится задействовать 'Babel' в качестве препроцессора.",
+        "Не забывайте пользоваться методом <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Читай-Ищи-Спрашивай</a>, если застрянете.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `если застрянете` sounds not so good. I suggest you to replace with something like this. `Если что-то не получается, не забывайте пользоваться` or  `воспользуйтесь методом <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Читай-Ищи-Спрашивай</a>`

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 5]
```diff
@@ -27,6 +27,20 @@
...
+        "Когда закончите, нажмите кнопку \"I've completed this challenge\" и укажите ссылку на вашу работу на CodePen.",
+        "Вы можете получить отзыв о вашем проекте от коллег, поделившись ссылкой на неё в нашем <a href='//gitter.im/freecodecamp/codereview' target='_blank'>Чате для Рассмотрения Кода</a>. Также вы можете поделиться ею через Twitter и в Кемпинге вашего города (на Facebook)."
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `неё` -> `него` everywhere.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 6]
```diff
@@ -67,6 +81,21 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте таблицу Кемперов-Лидеров",
+      "descriptionRu": [
+        "<strong>Задание:</strong> Создайте приложение <a href='http://codepen.io' target='_blank'>CodePen.io</a>, функционал которого схож с этим: <a href='http://codepen.io/FreeCodeCamp/full/qbqqJm/' target='_blank'>http://codepen.io/FreeCodeCamp/full/qbqqJm/</a>.",
+        "<strong>Правило #1:</strong> Не подсматривайте в код приложения-примера. Разберитесь самостоятельно.",
+        "<strong>Правило #2:</strong> Приложение должно удовлетворять нижеприведённым <a href='http://en.wikipedia.org/wiki/User_story' target='_blank'>пользовательским историям</a>. Используйте любые библиотеки или API, которые потребуются. Придайте ему свой личный стиль.",
+        "<strong>Правило #3:</strong> Для создания этого проекта вы должны использовать Sass и React.",
+        "<strong>Пользовательская История:</strong> Я могу видеть таблицу кемперов Free Code Camp, которые получили наибольшее количество очков брауни за прошедшие 30 дней.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `прошедшие` -> `последние` everywhere in this challenge.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 7]
```diff
@@ -67,6 +81,21 @@
...
+        "<strong>Пользовательская История:</strong> Я могу видеть сколько очков брауни они получили за прошедшие 30 дней, и сколько они получили их всего.",
+        "<strong>Пользовательская История:</strong> Я могу отсортировать список по количеству очков брауни, которые они получили за прошедшие 30 дней, и по общему количеству полученных очков брауни.",
+        "<strong>Подсказка:</strong> Для получения топ 100 кемперов за прошедшие 30 дней: <a href='http://fcctop100.herokuapp.com/api/fccusers/top/recent' target='_blank'>http://fcctop100.herokuapp.com/api/fccusers/top/recent</a>.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: This sentence sounds and feels not complete  while it is correct (there is no any verb to tell them what to do). I think that a verb like `воспользуйтесь` or `используйте` seems reasonable here.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 8]
```diff
@@ -67,6 +81,21 @@
...
+        "<strong>Подсказка:</strong> Для получения топ 100 кемперов за все время: <a href='http://fcctop100.herokuapp.com/api/fccusers/top/alltime' target='_blank'>http://fcctop100.herokuapp.com/api/fccusers/top/alltime</a>.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: The same thing here.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 9]
```diff
@@ -109,6 +138,22 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте Хранилище Рецептов",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `Создайте Хранилище Рецептов` -> `Создайте хранилище рецептов`

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 10]
```diff
@@ -109,6 +138,22 @@
...
+      "descriptionRu": [
+        "<strong>Задание:</strong> Создайте приложение <a href='http://codepen.io' target='_blank'>CodePen.io</a>, функционал которого схож с этим: <a href='http://codepen.io/FreeCodeCamp/full/LGbbqj' target='_blank'>http://codepen.io/FreeCodeCamp/full/LGbbqj</a>.",
+        "<strong>Правило #1:</strong> Не подсматривайте в код приложения-примера. Разберитесь самостоятельно.",
+        "<strong>Правило #2:</strong> Приложение должно удовлетворять нижеприведённым <a href='http://en.wikipedia.org/wiki/User_story' target='_blank'>пользовательским историям</a>. Используйте любые библиотеки или API, которые потребуются. Придайте ему свой личный стиль.",
+        "<strong>Правило #3:</strong> Для создания этого проекта вы должны использовать Sass и React.",
+        "<strong>Пользовательская История:</strong> Я могу создавать рецепты, содержащие название и ингредиенты.",
+        "<strong>Пользовательская История:</strong> Я могу просмотреть корневой вид, на котором видны все рецепты.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `корневой вид` sounds confusing but I don't have any other option now to suggest.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 11]
```diff
@@ -154,6 +199,24 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте Игру \"Жизнь\"",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `Создайте Игру \"Жизнь\"` -> `Создайте игру \"Жизнь\"

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 12]
```diff
@@ -154,6 +199,24 @@
...
+        "<strong>Правило #3:</strong> Для создания этого проекта вы должны использовать Sass и React.",
+        "<strong>Пользовательская История:</strong> Когда я впервые запускаю игру, она генерирует доску случайным образом и начинает игру.",
+        "<strong>Пользовательская История:</strong> Я могу запустить и остановить игру.",
+        "<strong>Пользовательская История:</strong> Я могу настроить доску.",
+        "<strong>Пользовательская История:</strong> Я могу очистить доску.",
+        "<strong>Пользовательская История:</strong> Когда я нажимаю начать, игра начинает воспроизведение.",
+        "<strong>Пользовательская История:</strong> Каждый раз, когда доска меняется, я могу видеть сколько поколений прошло.",
+        "<strong>Подсказка:</strong> Вот объяснение Игры \"Жизнь\" от её создателя Джона Конвея: <a href='https://www.youtube.com/watch?v=E8kUJL04ELA' target='_blank'>https://www.youtube.com/watch?v=E8kUJL04ELA</a>",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `объяснение Игры \"Жизнь\"` -> `объяснение игры \"Жизнь\"`

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 13]
```diff
@@ -154,6 +199,24 @@
...
+        "<strong>Подсказка:</strong> Вот обзор правил Игры \"Жизнь\" для вашего сведения: <a href='https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life' target='_blank'>https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life</a>",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `обзор правил Игры` -> `обзор правил игры`. Also I think it's better change Wiki link to the Russian translation: https://ru.wikipedia.org/wiki/Жизнь_(игра)

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 14]
```diff
@@ -202,6 +265,25 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте Рогулайк Данжеон-Кроулер",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: Roguelike is a game genre, no need to mention it (and even translate). Dungeon Crawler is the name of the game and there is no need to translate it. I even couldn't pronounce that is Russian )) So `Создайте игру Dungeon Crawler` id a much better option.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 15]
```diff
@@ -202,6 +265,25 @@
...
+      "descriptionRu": [
+        "<strong>Задание:</strong> Создайте приложение <a href='http://codepen.io' target='_blank'>CodePen.io</a>, функционал которого схож с этим: <a href='http://codepen.io/FreeCodeCamp/full/dGOOEJ/' target='_blank'>http://codepen.io/FreeCodeCamp/full/dGOOEJ/</a>.",
+        "<strong>Правило #1:</strong> Не подсматривайте в код приложения-примера. Разберитесь самостоятельно.",
+        "<strong>Правило #2:</strong> Приложение должно удовлетворять нижеприведённым <a href='http://en.wikipedia.org/wiki/User_story' target='_blank'>пользовательским историям</a>. Используйте любые библиотеки или API, которые потребуются. Придайте ему свой личный стиль.",
+        "<strong>Правило #3:</strong> Для создания этого проекта вы должны использовать Sass и React.",
+        "<strong>Пользовательская История:</strong> У меня есть жизни, уровень и оружие. Я могу подобрать лучшее оружие. Я могу подобрать очки здоровья.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `Я могу подобрать лучшее оружие` should be `Я могу подобрать оружие получше` because your translation means `I can pick up the best weapon` but it should mean `I can pick up a better weapon`.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 16]
```diff
@@ -202,6 +265,25 @@
...
+        "<strong>Пользовательская История:</strong> Все предметы и враги располагаются на карте случайным образом.",
+        "<strong>Пользовательская История:</strong> Я могу передвигаться по карте, обнаруживая новые предметы.",
+        "<strong>Пользовательская История:</strong> Я могу двигаться куда угодно в рамках карты, но не могу продвинуться дальше врага, пока он не будет побежден.",
+        "<strong>Пользовательская История:</strong> Большая часть карты скрыта. Когда я делаю шаг, все клетки в определенном количестве клеток от меня становятся видимы.",
+        "<strong>Пользовательская История:</strong> Когда я побеждаю врага, враг исчезает, а я получаю очки опыта (XP), набор которых приводит в результате к увеличению моего уровня.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: `набор которых приводит в результате к увеличению моего уровня` -> `что увеличивает мой уровень` or just remove `в результате` or place commas before and after `в результате`. But I think shorter === better.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 17]
```diff
@@ -27,6 +27,20 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте предпросмотрщик облегченного языка разметки Markdown",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: Remove the word `облегченного`. Yes, Markdown is defined as a lightweight markup language but I don't think we need to emphasize that in the title.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 18]
```diff
@@ -27,6 +27,20 @@
...
+      "descriptionRu": [
+        "<strong>Задание:</strong> Создайте приложение <a href='http://codepen.io' target='_blank'>CodePen.io</a>, функционал которого схож с этим: <a href='http://codepen.io/FreeCodeCamp/full/obYYqW' target='_blank'>http://codepen.io/FreeCodeCamp/full/obYYqW</a>.",
+        "<strong>Правило #1:</strong> Не подсматривайте в код приложения-примера. Разберитесь самостоятельно.",
+        "<strong>Правило #2:</strong> Приложение должно удовлетворять нижеприведённым <a href='http://en.wikipedia.org/wiki/User_story' target='_blank'>пользовательским историям</a>. Используйте любые библиотеки или API, которые потребуются. Придайте ему свой личный стиль.",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: Replace the Wiki link to the Russian translation of that article (here and everywhere): https://ru.wikipedia.org/wiki/Пользовательские_истории

**Comment by bugron** [comment]
@rfprod thanks for the translation. When you finish editing, [squash commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) into one and we'll be ready to merge.

File: seed/challenges/02-data-visualization-certification/react-projects.json (hunk 5 above)
Comment (rfprod) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: > ...`в Кемпинге вашего города (на Facebook).` sounds not so good. `на странице Facebook вашего города` is a better option.

Isn't `on the Facebook page of your city` misleading? E.g. if one is located in Minneapolis, then should link be posted here: https://www.facebook.com/cityofminneapolis/ ? Shouldn't Free Code Camp be mentioned somehow, what do you think?

File: seed/challenges/02-data-visualization-certification/react-projects.json (hunk 7 above)
Comment (rfprod) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: What do you think about: `Пример получения топ 100 кемперов за прошедшие 30 дней:` ?
The same sentence template can be used for next example of top100 of all time.

File: seed/challenges/02-data-visualization-certification/react-projects.json (hunk 5 above)
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: @rfprod OK, lets mention FCC, but `в Кемпинге` should be replaced.

File: seed/challenges/02-data-visualization-certification/react-projects.json (hunk 14 above)
Comment (rfprod) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: Dungeon Crawler does not have to be Roguelike. It also has a particular translation.
Roguelike - https://ru.wikipedia.org/wiki/Roguelike
Dungeon Crawler - https://ru.wikipedia.org/wiki/%D0%9F%D0%BE%D0%B4%D0%B7%D0%B5%D0%BC%D0%B5%D0%BB%D1%8C%D0%B5_%28%D1%80%D0%BE%D0%BB%D0%B5%D0%B2%D1%8B%D0%B5_%D0%B8%D0%B3%D1%80%D1%8B%29

so it should be, for example, `Создайте Рогулайк-игру Подземелье` or `Создайте Рогалик-игру Подземелье`, latter does not fit, mainly because there is an actual word `рогалик` with different meaning.

Are you sure, `Создайте игру Dungeon Crawler` should be used?

Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: What do you think about `Создайте игру Подземелье` without `Рогулайк`? Still I'm not sure why do we need to mention the genre. In that article you can find `Ранние roguelike` and `Roguelike-подобные игры` so in some places 'translated' version is used in some places original `roguelike`. In these cases I use originals.

File: seed/challenges/02-data-visualization-certification/react-projects.json (hunk 7 above)
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: That sounds better but it's not an actual `пример`, it's top 100 camper's info in JSON format.
What do you think about this; `Ссылка на топ 100 кемперов за последние 30 дней в формате JSON`?

Comment (rfprod) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: Sound most relevant, agreed.

File: seed/challenges/02-data-visualization-certification/react-projects.json (hunk 14 above)
Comment (rfprod) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: I will do whatever you decide at this point.
The argument for using `Рогулайк` in the task name is the fact that Dungeon Crawlers are not necessarily roguelike, and the difference is quite essential.
A random Reddit user knows that Dungeon Crawlers can be not roguelike: https://www.reddit.com/r/AndroidGaming/comments/2xvmy9/any_nonroguelike_dungeon_crawlers/
Also, you may want to check these links:
- http://www.roguebasin.com/index.php?title=What_a_roguelike_is
- http://www.roguetemple.com/roguelike-definition/

In conclusion, Make a Dungeon Crawler may seem clear for most people, but Make a Roguelike Dungeon Crawler narrows down the task, because one can search for particular term and read what should be done instead of just looking at the example and trying to figure out how it should work.

All other corrections are already made or will be made soon, so if you still insist on `Создайте игру Подземелье`, I'll use this phrase. Confirm it.

File: seed/challenges/02-data-visualization-certification/react-projects.json (hunk 5 above)
Comment (rfprod) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: Is `и на странице Free Code Camp вашего города на Facebook` fine from your point of view?

Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: That sounds much better.

File: seed/challenges/02-data-visualization-certification/react-projects.json (hunk 14 above)
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: I understand your point. But I don't like how they 'translated' the word (actually, a term) `Roguelike`. I'd use `Roguelike-подобную` instead of `Рогулайк` like this: `Создайте Roguelike-подобную игру Подземелье`.

**Comment by rfprod** [comment]
@bugron All changes are done, now I'm not sure what to do with this:
```
(add/RU-translation-react-projects) $ git rebase -i HEAD~n
fatal: Needed a single revision
invalid upstream HEAD~n
```
After making a final commit and preparing to squash commits before push, I discovered that I did not do `git pull --rebase upstream staging` before making changes.

Provide further instructions, please.

**Comment by bugron** [comment]
@rfprod we can pair by Screenhero if you want.

**Comment by rfprod** [comment]
@bugron Screenhero does not accept new signups according to statement on the [login page](https://screenhero.com/login/). I created slack.com account, but the are offering Screenhero to [paid accounts](https://fcc-ru-translation.slack.com/screenhero) only. And I'm not really comfortable with anyone gaining control over my computer in general. If you provide any helpful links, it will be enough, I think.

**Comment by rfprod** [comment]
Ok, it seems commits are squashed:
```
[detached HEAD 26dd3d9] Russian Translation of React Projects. Closes #6808
 1 file changed, 82 insertions(+)
Successfully rebased and updated refs/heads/add/RU-translation-react-projects.
```

**Comment by bugron** [comment]
@rfprod OK, now just force push your branch to your fork. `git push -f YOUR_UPSTREAM YOUR_BRANCH`.

File: seed/challenges/02-data-visualization-certification/react-projects.json [hunk 19]
```diff
@@ -27,6 +27,20 @@
...
+        "<strong>Пользовательская история:</strong> Я могу видеть предварительный просмотр вывода моей разметки по мере ввода текста.",
+        "<strong>Подсказка:</strong> Вам не нужно интерпретировать разметку самостоятельно - вы можете импортировать библиотеку Marked для этого: <a href='https://cdnjs.com/libraries/marked'>https://cdnjs.com/libraries/marked</a>",
+        "<strong>Заметка:</strong> Если вы хотите использовать синтаксис React JSX, вам понадобится задействовать 'Babel' в качестве препроцессора.",
+        "Если что-то не получается, не забывайте пользоваться методом <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Читай-Ищи-Спрашивай</a>.",
+        "Когда закончите, нажмите кнопку \"I've completed this challenge\" и укажите ссылку на вашу работу на CodePen.",
+        "Вы можете получить отзыв о вашем проекте от коллег, поделившись ссылкой на него в нашем <a href='//gitter.im/freecodecamp/codereview' target='_blank'>чате для рассмотрения кода</a>. Также вы можете поделиться ею через Twitter и на странице Free Code Camp вашего города на Facebook."
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: Now everything looks really good. The only thing is that in `Также вы можете поделиться ею` the usage of `ею` is incorrect because the word `проект` is masculine. It should be `им` here and everywhere.

Comment (rfprod) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: In fact it is correct, because `ею` points on the link to the project, i.e. word `ссылкой`, and `ссылка` is feminine.

Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: In that case yes, it is. I was not sure are you talking about the word `проект` or `ссылка` because you can both `поделиться проектом` and `поделиться ссылкой`, thanks for clarifying.

Comment (rfprod) [inline seed/challenges/02-data-visualization-certification/react-projects.json]: @bugron you're always welcome.

**Comment by bugron** [comment]
@rfprod thank you for this awesome translation. It's really great. Would love to QA more translation PRs from you. And thank you for your patience :)
Verified and merged!

**Comment by rfprod** [comment]
@bugron I will make another translation after current backend challenge is finished. This will happen soon. Thank you for your assistance.
